src/components/layout/Header.tsx

```diff
@@ -19,28 +25,41 @@ export default function Header() {
         <ul className={styles.navList}>
           <li>
             <Link
-              className={path === "/boards" ? styles.navItem_active : undefined}
+              className={
+                pathname === "/boards" ? styles.navItem_active : undefined
+              }
               href="/boards"
             >
               자유게시판
             </Link>
           </li>
           <li>
             <Link
-              className={path === "/items" ? styles.navItem_active : undefined}
+              className={
+                pathname === "/items" ? styles.navItem_active : undefined
+              }
               href="/items"
             >
               중고마켓
             </Link>
           </li>
         </ul>
       </nav>
-      {token ? (
-        <Image
-          className={styles.profile}
-          src={profileImage}
-          alt="프로필 사진"
-        />
+      {isLoading ? undefined : user ? (
```

Comment:
P2:
이중 삼항연산자는 가독성이 떨어지니 조건을 더 단순화하거나 분리하시는 것이 좋을 것 같습니다.
만약 isLoading일 시 undefined가 반환되야 한다면, 즉 아무것도 렌더링될 필요가 없다면 아래 조건도 가능할 것 같습니다.
```suggestion
      {!isLoading && user ? (
```